hc/settings.py

```diff
@@ -120,8 +93,8 @@
 
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
-SITE_NAME = "healthchecks.io"
+SITE_ROOT = "https://hc-bs-staging.herokuapp.com/"
```

Comment:
It is on this branch. Otherwise, I would not comment on it from this PR. I am referring to `SITE_ROOT` [here](https://github.com/andela/healthchecks-boomslang/blob/20d403039b988a12751be167e7d288c3afc08a3c/hc/settings.py#L96)